addon-test-support/properties/attribute.js

```diff
@@ -82,9 +83,11 @@ export function attribute(attributeName, selector, userOptions = {}) {
     get(key) {
       let options = assign({ pageObjectKey: key }, userOptions);
 
-      let elements = findElementWithAssert(this, selector, options);
-
-      return map(elements, element => element.attr(attributeName), options);
+      if (options.multiple) {
```

Comment:
I know this makes total sense for backward compatibility. However..

I believe we should discourage users from passing `multiple` options for attributes. 
Such usages can/should be expressed with collections IMO, which would:
 - reduce API surface
 - simplify interanals, so each propoerty is always associated with a single node
 - lead to a more consistent page object definitions

so in my opinion, as the next step we should deprecate this behavior.

Reply:
💯 May be we should add a deprecation message whenever it's used?

Reply:
Yes, but I think we should keep it separate from this PR